lib/workers/repository/update/pr/code-owners.ts

```diff
@@ -21,37 +32,129 @@ export async function codeOwnersForPr(pr: Pr): Promise<string[]> {
 
     logger.debug(`Found CODEOWNERS file: ${codeOwnersFile}`);
 
+    // Get list of modified files in PR
     const prFiles = await getBranchFiles(pr.sourceBranch);
-    const rules = codeOwnersFile
+
+    if (!prFiles) {
```

Comment:
```suggestion
    if (!prFiles?.length) {
```
so we stop for empty list too